Sprint 8-Array&Loops/tema4.js

```diff
@@ -0,0 +1,15 @@
+// Returnează numărul de vocale dintr-un șir de caractere (string)
+
+const text = "string de test";
+
+let nr = 0;
```

Comment:
initializarea lui nr trebuie sa se afle in interiorul functiei.

Reply:
done